### ISSUE TITLE
Docker image / BusyBox version

### ISSUE THREAD

**Issue by ddewaele** [prometheus/prometheus#3161]
We've centralized our prometheus / alertmanager config via a Spring Cloud Config (http) server.

We've extended the prom/prometheus image to fetch the config before starting prometheus. 

We're having issues when we are retrieving the config through an Amazon Application Loader Balancer, where wget returns `error getting response`

This seems to be related to the following BusyBox issue, fixed in v1.27 https://bugs.busybox.net/show_bug.cgi?id=9471

I noticed that the Prometheus Dockerfile uses `quay.io/prometheus/busybox:latest`, a v1.26.2 version.

Here are the results of the wget call : 

Prometheus v1.7.1
```
docker run -ti --rm prom/prometheus:1.7.1

Linux caea1378173d 4.9.41-moby #1 SMP Wed Sep 6 00:05:16 UTC 2017 x86_64 GNU/Linux
BusyBox v1.26.2 (2017-01-12 18:09:33 UTC) multi-call binary.
/ #  wget http://aws-app-loadbalancer/config/ixortalk.prometheus/dev/master/prometheus.yml
wget: error getting response
```

Using the latest busybox the wget works fine:
```
docker run -it --rm busybox:1.27

Linux cbbf44617185 4.9.41-moby #1 SMP Wed Sep 6 00:05:16 UTC 2017 x86_64 GNU/Linux
BusyBox v1.27.2 (2017-08-22 23:59:28 UTC) multi-call binary.

/ #  wget http://aws-app-loadbalancer/config/ixortalk.prometheus/dev/master/prometheus.yml
prometheus.yml       100% 
```

Are there any plans have prometheus use a more up-to-date version of BusyBox ?

**Comment by sdurrheimer** [comment]
I have trigger a new build of the `quay.io/prometheus/busybox:latest` which will then use the latest busybox version.

But this won't rebuild the prometheus 1.7.1 release on this updated image.

**Comment by ddewaele** [comment]
ok .... will `prometheus:latest` then pick up the new busybox version ?

**Comment by sdurrheimer** [comment]
No, `prometheus:latest` correspond to the latest stable release, which is `1.7.1` currently.

You will need to wait a `1.7.2` release or try the `master` tag.

**Comment by ddewaele** [comment]
You guys rock ! the wget works now ... will do some additional testing with prometheus itself.

So if all goes well the 1.7.2 release will contain BusyBox 1.27 ? 

Is there a timeline yet ?

**Comment by ddewaele** [comment]
Also noticed that prometheus and alertmanager are using different busy box base images: 

- Prometheus : quay.io/prometheus/busybox:latest
- AlertManager : prom/busybox:latest

Perhaps these should also be aligned ?

**Comment by sdurrheimer** [comment]
> So if all goes well the 1.7.2 release will contain BusyBox 1.27 ?
Is there a timeline yet ?

Yes it will but I don't know when a 1.7.2 will be released. Maybe @fabxc or @brancz have more informations about that.

> Also noticed that prometheus and alertmanager are using different busy box base images:
Prometheus : quay.io/prometheus/busybox:latest
AlertManager : prom/busybox:latest
Perhaps these should also be aligned ?

You're right, this should be aligned, but technically it is the same image, just pushed to two different repository.

**Comment by brancz** [comment]
I'm happy to cut a release if reasonable.
